### PR TITLE
📝 add a gif showing centerpiece usage

### PR DESCRIPTION
**Pull request by friedow**
Big thanks to @a-kenji

### CODE REVIEW

File: screenshots/demo.webm
Comment (a-kenji) [inline screenshots/demo.webm:1]: Are we still interested in the `webm` file itself?

**Review by a-kenji — COMMENTED** [review]
LGTM

File: screenshots/demo.webm
Comment (friedow) [reply screenshots/demo.webm:1]: I think it's a better source file in case we want to reuse this video on another case.

Comment (a-kenji) [reply screenshots/demo.webm:1]: That makes sense, thanks for the explanation :+1: .

**Pull request merged**